<?xml version="1.0" encoding="UTF-8"?>
<bajadoc version="2.0" createdBy="niagara-baja-doclet-1.0.9" createdAt="03-May-2024" createdOn="883e7f7a9875">
<class module="history" runtimeProfile="rt" qualifiedName="javax.baja.history.DuplicateHistoryException" name="DuplicateHistoryException" packageName="javax.baja.history" public="true" category="exception">
<description>
DuplicateHistoryException is thrown when an attempt&#xa; is made to create a history with an id that already exists.
</description>
<tag name="@author">John Sublett</tag>
<tag name="@creation">18 Jul 2002</tag>
<tag name="@version">$Revision: 1$ $Date: 4/4/03 5:21:09 PM EST$</tag>
<tag name="@since">Baja 1.0</tag>
<extends>
<type class="javax.baja.history.HistoryException"/>
</extends>
<!-- javax.baja.history.DuplicateHistoryException(javax.baja.history.BHistoryId) -->
<constructor name="DuplicateHistoryException" public="true">
<parameter name="id">
<type class="javax.baja.history.BHistoryId"/>
</parameter>
<description>
Construct an instance with the specified history id.
</description>
</constructor>

<!-- javax.baja.history.DuplicateHistoryException(java.lang.String) -->
<constructor name="DuplicateHistoryException" public="true">
<parameter name="msg">
<type class="java.lang.String"/>
</parameter>
<description>
Construct an instance with the given message.
</description>
</constructor>

<!-- javax.baja.history.DuplicateHistoryException() -->
<constructor name="DuplicateHistoryException" public="true">
<description>
No argument constructor.
</description>
</constructor>

</class>
</bajadoc>
